core/header/service.go

```diff
@@ -0,0 +1,17 @@
+package header
```

Comment:
Is this descriptive enough. Let's add docs

Reply:
added a doc.go's to the package